themes/sensei-course-theme/single.php

```diff
@@ -45,7 +45,8 @@
 
 	<!-- wp:column {"width":"","className":"sensei-course-theme__main-content"} -->
 	<div class="wp-block-column sensei-course-theme__main-content">
-		<!-- wp:post-title /-->
+		<!-- wp:sensei-lms/course-theme-module-title /-->
+		<!-- wp:sensei-lms/course-theme-lesson-title /-->
```

Comment:
Not sure why we would want a title block that only works for lessons. We will have quiz and module pages using this template too. 

The core `post-title` block would be fine here — the only issue is it's not present on all our supported Wordpress versions, so we'll need our own version for now. Let's do a `sensei-lms/post-title` that just outputs the current post title. 
Also see the [courses-title](https://github.com/Automattic/sensei/blob/443f779af4c9cd7df3d6131c2477efc9c2a6a5aa/includes/blocks/course-theme/class-course-title.php#L56) block that includes support for controlling the h1-h6 levels and other attributes from the template (and later the editor).

Reply:
I renamed it to be `sensei-lms/course-theme-post-title` and modified it to render title of the post that is returned by `get_the_ID()`. Does that sound good?